mesonpy/__init__.py

```diff
@@ -102,7 +107,7 @@ def _init_colors() -> Dict[str, str]:
 _STYLES = _init_colors()  # holds the color values, should be _COLORS or _NO_COLORS
 
 
-_EXTENSION_SUFFIXES = frozenset(importlib.machinery.EXTENSION_SUFFIXES)
+_EXTENSION_SUFFIXES = list(importlib.machinery.EXTENSION_SUFFIXES)
```

Comment:
`importlib.machinery.EXTENSION_SUFFIXES` is already a `list`. No coercion is necessary here.

Reply:
Why do we need a copy at all?